2_process.R

```diff
@@ -88,72 +90,143 @@ p2_targets_list <- list(
   # returns a df with unique comids for aoi + area of comid and NLCD LC percentage attributes
   tar_target(
     p2_NLCD_LC_w_catchment_area,
-    AOI_LC_w_area(area_att = p1_nhdv2reaches_sf %>% st_drop_geometry() %>% select(COMID,AREASQKM,TOTDASQKM,LENGTHKM),
-                  ## NOTE - the NLCD_LC_df selected in the Land Cover 2011 - to be looped across all items of p1_NLCD_data
-                  NLCD_LC_df = p1_NLCD_data$NLCD_LandCover_2011,
+    AOI_LC_w_area(area_att = p1_nhdv2reaches_sf %>% 
+                                                st_drop_geometry() %>% 
+                                                select(COMID, AREASQKM,TOTDASQKM, LENGTHKM),
+                  NLCD_LC_df = p1_NLCD_LC_data,
                   aoi_comids_df = p2_drb_comids_all_tribs)
   ),
   
-  ## Estimate LC proportion in PRMS catchment
+  ## Estimate LC proportion in PRMS catchment - CAT, TOT, and ACC
   # returns df with proportion LC in PRMS catchment in our AOI
-  tar_target(p2_PRMS_NLCD_lc_proportions,
-             proportion_lc_by_prms(p2_NLCD_LC_w_catchment_area)),
-
+  tar_target(
+    p2_PRMS_NLCD_lc_proportions_cat,
+             proportion_lc_by_prms(p2_NLCD_LC_w_catchment_area,
+                                   catchment_att = 'CAT') %>%
+               select(-contains('NODATA'))),
+  
+  tar_target(
+    p2_PRMS_NLCD_lc_proportions_tot,
+    proportion_lc_by_prms(p2_NLCD_LC_w_catchment_area %>%
+                            # filtering to only the comid_downs of each PRMS - nrow = ~459
+                            filter(comid %in% p2_drb_comids_down$comid),
+                          catchment_att = 'TOT') %>%
+      select(-contains('NODATA'))),  
+  
+  tar_target(
+    p2_PRMS_NLCD_lc_proportions_acc,
+    proportion_lc_by_prms(p2_NLCD_LC_w_catchment_area %>%
+                            # filtering to only the comid_downs of each PRMS - nrow = ~459
+                            filter(comid %in% p2_drb_comids_down$comid),
+                          catchment_att = 'ACC') %>%
+      select(-contains('NODATA'))), 
```

Comment:
can you fix the indentation of the `select` functions? And also add a return before the `),` at the end of each target. Example of the `),`: 
```
select(-contains('NODATA'))
),
```